src/render.tsx

```diff
@@ -224,13 +230,38 @@ const isAnchor = (content: HtmlContent | HtmlElement): boolean => {
     : content.tag === 'a';
 };
 
+const isSvgText = (content: HtmlElement | undefined): boolean => {
+  return (
+    [
+      'svg',
+      'line',
+      'polyline',
+      'polygon',
+      'path',
+      'rect',
+      'circle',
+      'ellipse',
+      'text',
+      'tspan',
+      'g',
+      'stop',
+      'defs',
+      'clippath',
+      'lineargradient',
+      'radialgradient',
+    ].indexOf(content?.tag || '') > -1
```

Comment:
This could use ‘contains’ instead of ‘index Of’

Reply:
funny, I tried contains at first and gave up and used indexOf. Turns out we were looking for includes.